Main/ConsistencyCheck.cpp

```diff
@@ -88,6 +89,8 @@ ConsistencyCheck::CheckForDuplicates()
 		if (name1->Name().Compare(name2->Name()) == 0) {
 			name1->SetDuplicate();
 			name2->SetDuplicate();
+			duplicateFound = true;
 		}
 	}
+	return duplicateFound;
```

Comment:
I think you can do this without creating duplicateFound.  Just return true on line 92, or return false on 95.